client/src/locales/en-US.json

```diff
@@ -51,7 +51,7 @@
     },
     "notificationCenter": {
         "title": "Notifications",
-        "readOnly": "Only no read",
+        "readOnly": "Only unread",
```

Comment:
Maybe we could, but I'm not sure it would be as clear as "only unread".
![image](https://github.com/Scille/parsec-cloud/assets/19772607/a515b95f-d63b-44f1-bdfb-816e4199223d)